src/common/utils/log/beegoadapter.go

```diff
@@ -0,0 +1,90 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/astaxie/beego/logs"
+)
+
+// BeeLevel2Hbr mapping beego's log level to harbor's log lever
```

Comment:
Do we have to export this variable?

Reply:
I think import logs are the most convenient way or I need using integers as the key which is not good for readability.